volume/local/local.go

```diff
@@ -333,18 +331,6 @@ func (v *localVolume) Unmount(id string) error {
 	return v.unmount()
 }
 
-func (v *localVolume) unmount() error {
-	if v.opts != nil {
-		if err := mount.Unmount(v.path); err != nil {
```

Comment:
same question here

Reply:
No idea on this one (why it didn't panic on Windows); I guess because we're skipping related tests on Windows; https://github.com/moby/moby/blob/af0415257efa288d9444de46ba2b45868c7c1a54/volume/local/local_test.go#L33-L34

Wondering if a combination of this, and https://github.com/moby/moby/pull/40599 lies at the root of that